.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
It’s not related to project’s code, otherwise it might fill with VS Code, Sublime, Atom, Notepad++, etc. configs. There’s [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) for that.